### DISCUSSION TITLE
[v.0.5.0] New logo used for shared folder splash screen

### DISCUSSION THREAD

**Discussion by leonarzoh** [radio24/TorBox#117] [Archive]
Hi all. Just would like to share a new logo proposal with everyone, as suggested by @radio24.
Basically, it's a purple box using the onion icon. I can create alternative versions, perhaps using green or something like that.  I have the vector version (svg, ai etc) on Figma, so I can share the file if someone would like to fork it. 

![1200-414-max](https://user-images.githubusercontent.com/85939670/124350192-13bec900-dbeb-11eb-854f-52008f3ff863.png)

**Comment by radio24** [comment]
I reopen this contribution from [leonarzoh](https://github.com/leonarzoh). 
Probably, I will use it for the splash screen shown when a folder is shared through Onion Services, integrated with the next version of TorBox.

**Comment by radio24** [comment]
Coming soon ....
<img width="794" alt="splash_screen_shared_folders" src="https://user-images.githubusercontent.com/46644350/142694649-5597ee43-66c8-4a69-b0db-c8e27082a172.png">

**Reply by nyxnor** [reply]
Beautiful.
Little typo: Congratulation`s`

**Reply by radio24** [reply]
Thanks, it will be fixed in the final version.

**Comment by nyxnor** [comment]
wowwww
